selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java

```diff
@@ -288,9 +289,10 @@ public void addDrives(String name, String permission, specificView[] applyViews)
    *          other will be uncheck)
    */
   public void editDrives(String name, specificView[] applyViews, String... views) {
-    $(byClassName("actionIcon")).click();
+    executeJavaScript("window.scrollBy(0,-250);");
```

Comment:
yes, we put Y as negative value for scroll up and positive value for scroll down (Y is the number of pixels )